app/models/filters/filter_base.rb

```diff
@@ -69,6 +69,7 @@ class FilterBase < ::ModelForm
     attribute :report_version, Symbol
     attribute :inactivity_days, Integer, default: 365 * 2
     attribute :lsa_scope, Integer, default: nil
+    attribute :youth, Boolean, default: false
```

Comment:
Is there some reason we couldn't just use the age ranges for this?

Reply:
I could be convinced -- I just saw that the spec says to only include <= 24 y/o, so, having both that and the age ranges seemed confusing... We could use age ranges, and just have 0 to 24 and 18 to 24 as the only 2 options...

Reply:
I think there's a 12-24 or 12-18 category that we should allow as well.

Reply:
Basically any category where the max is 24 or less

Reply:
The relevant options in in `FilterScopes` are _under 18_, and _18-24_. The weirdness for using the defaults, is that we would need to default to 0-24 as the option -- that is what the report expects -- and that either means adding overlapping option (which I don't like), or selecting both of the ranges, which would look weird... I guess we could override the display, and not show *Age Ranges* if they are both selected.

Reply:
ah, I guess we did the 12-18 somewhere else.  I'd just make it so you can only pick from these two, and they are both selected by default:

```
under_eighteen: '< 18',
eighteen_to_twenty_four: '18 - 24',
```